packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -935,8 +955,8 @@ export type DownloadCsv = BaseTrack & {
         context?:
```

Comment:
we should probably standarize this a bit more and reuse the QueryExecutionContext type